DEPS

```diff
@@ -364,7 +364,7 @@ deps = {
    Var('dart_git') + '/package_resolver.git' + '@' + Var('dart_package_resolver_tag'),
 
   'src/third_party/dart/tools/sdks':
-   {'packages': [{'version': 'version:2.3.3-dev.0.0', 'package': 'dart/dart-sdk/${{platform}}'}], 'dep_type': 'cipd'},
```

Comment:
Was this change needed for the API change?

Reply:
It doesn't appear to be in the reverted isolate group change (https://github.com/flutter/engine/commit/64b9eef5c1671a4670e012350830b508001becb3) but that wasn't a change I made manually. It looks like that was a change made in the Dart SDK DEPS today by Alex.